package.json

```diff
@@ -49,7 +49,12 @@
         "markdown-links.fileIdRegexp": {
           "type": "string",
           "default": "\\d{14}",
-          "description": "Regular extension used to find file IDs. First match of this regex in file contents, excluding [[links]], will be used as the file ID. This file ID can be used for wiki-style links."
+          "description": "Regular expression used to find file IDs. First match of this regex in file contents, excluding [[links]], will be used as the file ID. This file ID can be used for wiki-style links."
+        },
+        "markdown-links.searchGlob" : {
```

Comment:
Once we release this PR, `searchGlob` is going to be a part of the public interface and very hard to rename in the future. I'm open to any other names like `fileSearchGlob` `findFilesExpression` or any cobination of the `search`, `find`, `files`, `expression`, `glob`..

Reply:
Very good idea. Appreciate a lot the fact that you consider it already at this point!

I think `searchGlob` is ok. Maybe it would be more future proof to specify what is the search we are doing here. Maybe `fileDiscoveryGlob` presents it better?

Reply:
I went with `findFilesGlob` for consistency. Let me know if that's ok :)